openhtf/io/output/mfg_inspector.py

```diff
@@ -334,13 +326,11 @@ def _TestRunFromTestRecord(record):
   return testrun
 
 
-class OutputToTestRunProto(object):  # pylint: disable=too-few-public-methods
+class OutputToTestRunProto(output.OutputToFile):  # pylint: disable=too-few-public-methods
```

Comment:
I really like the simplification of the subclasses that this change enables, nice job with it!